chevah/brink
# [#3332] Fix running elevated tests with normal python test runner.

Scope
=====

This fixes the way tests are selected to be executed on buildslaves


Changes
=======

I have fixed the way tests are selected when arguments are provided.

As a drive by, added a test_diff helper which will run only tests which 

Version 0.66.0 was already released.


How to try and test the changes
===============================

reviewers: @hcs0 

This is used mostly for developers... but here were are :)

Check that you can select remote tests, just like you did on local.

Any `paver test [ARGS]` should be able to be replaced with `paver test_remote ubuntu-1604 --wait [ARGS]` and the same tests should be selected to be executed

```
# Both normal and elevated tests are executed with listing the names of the tests which were executed
$ paver test_remote ubuntu-1604 --wait -v

# Only the 2 elevated test are executed.
$ paver test_remote ubuntu-1604 --wait elevated.test_filesystem:TestBrinkFilesystem -v

# A single normal tests and a single elevated tests is executed 
$ paver test_remote ubuntu-1604 --wait elevated.test_filesystem:TestBrinkFilesystem.test_placeholder test_filesystem:TestBrinkFilesystem.test_parseWindowsPaths -v
```
